src/plugins/workspace/server/saved_objects/workspace_ui_settings_client_wrapper.ts

```diff
@@ -61,9 +63,23 @@ export class WorkspaceUiSettingsClientWrapper {
           options
         );
 
-        const workspaceObject = await this.getWorkspaceTypeEnabledClient(
-          wrapperOptions.request
-        ).get<WorkspaceAttribute>(WORKSPACE_TYPE, requestWorkspaceId);
+        let workspaceObject: SavedObject<WorkspaceAttribute> = {
+          id: requestWorkspaceId,
+          type: 'workspace',
+          attributes: {
+            id: requestWorkspaceId,
+            name: '',
+          },
+          references: [],
+        };
+
+        try {
+          workspaceObject = await this.getWorkspaceTypeEnabledClient(wrapperOptions.request).get<
+            WorkspaceAttribute
+          >(WORKSPACE_TYPE, requestWorkspaceId);
+        } catch (e) {
+          this.logger.error(`Unable to get workspaceObject on id: ${requestWorkspaceId}`);
+        }
```

Comment:
Instead of giving `workspaceObject` a dummy empty value, what about just initialize it as `null`?

```suggestion
        let workspaceObject: SavedObject<WorkspaceAttribute> | null = null

        try {
          workspaceObject = await this.getWorkspaceTypeEnabledClient(wrapperOptions.request).get<
            WorkspaceAttribute
          >(WORKSPACE_TYPE, requestWorkspaceId);
        } catch (e) {
          this.logger.error(`Unable to get workspaceObject on id: ${requestWorkspaceId}`);
        }
        
        configObject.attributes = {
          ...configObject.attributes,
          ...(workspaceObject ? workspaceObject.attributes.uiSettings : {}),
        };
```

Reply:
Sure, it looks more concise.